framework/resources/Functional/filter/pushdown/varchar_decimal/hive/varchar_decimal_hive_hive.plan.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
The file name has a redundant word ..hive_hive..

Reply:
Fixed.